<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mesero_tarjeta_chica</class>
 <widget class="QWidget" name="mesero_tarjeta_chica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>235</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
border: 1px solid #ECEEEE;
}

QLabel {
border:none;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>235</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: #fff;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="imgPlatillo">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/iconos/resources/menuPrueba.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../Recursos.qrc">:/iconos/resources/menuPrueba.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="nombrePlatillo">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#585858;
</string>
         </property>
         <property name="text">
          <string>Sopa de Codito</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="labelCantidad">
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#858585;
</string>
           </property>
           <property name="text">
            <string>Cant:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cantidad">
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#858585;
</string>
           </property>
           <property name="text">
            <string>#</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="precioPlatillo">
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#858585;
</string>
           </property>
           <property name="text">
            <string>$75.00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
